enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/websocket/WebSocketMetaDataFilter.java

```diff
@@ -74,8 +74,6 @@ public void init(APIConfig apiConfig, Map<String, String> configProperties) {
                     getNullableStringValue(authenticationContext.getApplicationTier()));
             requestContext.addMetadataToMap(MetadataConstants.TIER,
                     getNullableStringValue(authenticationContext.getTier()));
-            requestContext.addMetadataToMap(MetadataConstants.API_TIER,
-                    getNullableStringValue(authenticationContext.getApiTier()));
```

Comment:
We need to populate metadata somehow. Hence we need to populate it using the APIConfig.

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/2496/commits/d004fc72875effcfcd566a555a9f2c2117b67d14